recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ build:
   number: 0
   skip: True  # [osx]
   skip: True  # [cuda_compiler_version == "11.2"]
```

Comment:
We can drop this as [CUDA 11.2 was already dropped]( https://conda-forge.org/news/2024/03/06/dropping-cuda-112/ )

Reply:
```suggestion
```